### ISSUE TITLE
Paginator: add arial-label to JumpToPage components

### ISSUE THREAD

**Issue by JuanMiguelBG-Ticarum** [primefaces/primefaces#11771]
### Describe the bug

On paginator component, the JumpToPage components lacks of aria-label.

### PrimeFaces edition

None

### PrimeFaces version

13.0.x

### JSF implementation

Mojarra

### JSF version

2.2

### Java version

1.8

**Comment by JuanMiguelBG-Ticarum** [comment]
Yes, I know but the PR is target on 13.x branch.

**Comment by melloware** [comment]
Hmmm let me think about this.

**Comment by JuanMiguelBG-Ticarum** [comment]
The PR is wrong, they are not the changes I wanted to upload.

**Comment by melloware** [comment]
Just let me know when you want me to review it.

**Comment by JuanMiguelBG-Ticarum** [comment]
The right PR: [11779](https://github.com/primefaces/primefaces/pull/11779)
